cmd/cli/job/executions.go

```diff
@@ -147,7 +148,7 @@ func (o *ExecutionOptions) run(cmd *cobra.Command, args []string, api client.API
 		},
 	})
 	if err != nil {
-		return err
+		return errors.New(err.Error())
```

Comment:
please revert this, either do:
`return err`
or 
`return fmt.Errorf("<helpful message>: %w", err)`